fp-course.cabal

```diff
@@ -5,7 +5,7 @@ version:              0.1.4
 build-type:           Simple
 extra-source-files:   etc/CONTRIBUTORS
 
-license:              BSD3
+license:              BSD-3-Clause
```

Comment:
Cabal complains about `BSD3`, see: https://github.com/haskell/cabal/issues/5109

May want to pick this up in your fork @friedbrice.

Reply:
I saw that! Thanks for bringing it up.